client/app/states/services/custom_button_details/custom_button_details.state.spec.js

```diff
@@ -122,4 +122,80 @@ describe('services.custom_button_details', function() {
       });
     });
   });
+  
+  describe('Custom button actions for a VM', () => {
+    var collectionsApiSpy;
```

Comment:
prefer, const or let to var. consider stacking all the ones without assignments on the same line